watergun_assassin/authentication/tests.py

```diff
@@ -0,0 +1,54 @@
+from django.test import TestCase
+from django.urls import reverse
+from datetime import datetime
+
+
+class UserCreationTest(TestCase):
+    def test_valid_payload(self):
+        url = reverse('create_user')
+        username = 'admin'
+        password = 'kjlkjlkjlk123'
+        email = 'email@company.com'
+        data = {'username':username,'password':password,'email':email}
```

Comment:
We should get a code formatter in place to automatically format files when they are saved, but until we do let's establish a baseline. Can we always put spaces after commas? It makes things a bit more readable.